package.json

```diff
@@ -16,16 +19,21 @@
     "jest": "^29.4.1",
     "ts-jest": "^29.0.5",
     "tslib": "^2.5.0",
-    "typescript": "4.2.3"
+    "typescript": "^5.0.4"
   },
   "dependencies": {
     "axios": "^1.3.2",
+    "https-browserify": "^1.0.0",
     "stellar-sdk": "^10.4.1",
-    "ts-loader": "8.0.7",
-    "webpack": "4.41.5",
-    "webpack-cli": "3.3.10"
+    "stream-http": "^3.2.0",
+    "ts-loader": "^9.4.2",
+    "url": "^0.11.0",
+    "util": "^0.12.5",
+    "webpack": "^5.83.1",
+    "webpack-cli": "^5.1.1"
```

Comment:
it appears the main issue was that we needed to upgrade webpack v5, which fixes some security vulnerabilities. Because of this, we will need to mandate that users use node >=18

stream-http, url, util, and https no longer come prepackaged with webpack as of v5, so we need to add those manually.

I also upgraded Typescript because why not?

Reply:
Awesome, thanks for doing this!

Reply:
You can move a ton of these into devDependencies!